src/common/NetworkEventManager.ts

```diff
@@ -1,7 +1,7 @@
 import { Protocol } from 'devtools-protocol';
 import { HTTPRequest } from './HTTPRequest.js';
 
-export type QueuedEvents = {
+export type QueuedEventGroup = {
   responseReceived: Protocol.Network.ResponseReceivedEvent;
   promise: Promise<void>;
   resolver: () => void;
```

Comment:
From what I could find, `promise` and `resolver` are initialized but never actually awaited or referenced anywhere. I think these fields and the corresponding logic in `NetworkManager` can be removed.

Reply:
I think both are used in NetworkManager.